tests/intern-browserstack.ts

```diff
@@ -0,0 +1,14 @@
+export * from './intern';
```

Comment:
These files are turned into JSON using a convoluted process in `grunt-dojo2`, you can see that here: https://github.com/dojo/grunt-dojo2/pull/146